custom_components/plugwise/select.py

```
@@ -83,28 +83,21 @@ async def async_setup_entry(
-        # Upstream consts
-        # async_add_entities(
-        #     PlugwiseSelectEntity(coordinator, device_id, description)
-        #     for device_id in coordinator.new_devices
-        #     for description in SELECT_TYPES
-        #     if description.options_key in coordinator.data.devices[device_id]
-        # )
-        # pw-beta alternative for debugging
-        entities: list[PlugwiseSelectEntity] = []
-        for device_id in coordinator.new_devices:
-            device = coordinator.data.devices[device_id]
+        entities: list[PlugwiseClimateEntity] = []
```

Ruff

Undefined name `PlugwiseClimateEntity`

(F821)